README.md

```diff
@@ -146,7 +182,35 @@ class SomeController {
 
 ### Directive
 
-Static class members of directive controller are used as config directive config.
+You can configure your directive with annotation :
+
+```typescript
+@directive('ngModuleName', {
+    selector:     'atSomeDirective',                //required
+    controllerAs: 'someDirectiveCtrl',              //optional
+    templateUrl:  '/partials/some-directive.html'
+})
+class SomeDirectiveController {
+
+    public static link: angular.IDirectiveLinkFn = (scope, element, attrs, ctrl: SomeDirectiveController) => {
```

Comment:
this one here mixes 2 approaches, maybe for clarity of README move it to `@directive` annotation